packages/cli/tests/e2e/bot/ProvisionAppServiceNotificationBot.tests.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @author Xiaofu Huang <xiaofhua@microsoft.com>
+ */
+
+import fs from "fs-extra";
+import path from "path";
+
+import { AadValidator, BotValidator } from "../../commonlib";
+
+import {
+  execAsync,
+  execAsyncWithRetry,
+  getSubscriptionId,
+  getTestFolder,
+  getUniqueAppName,
+  cleanUp,
+  readContextMultiEnv,
+} from "../commonUtils";
+import AppStudioLogin from "../../../src/commonlib/appStudioLogin";
+import { environmentManager } from "@microsoft/teamsfx-core";
+
+import { it } from "../../commonlib/it";
+
+describe("Provision", function () {
+  const testFolder = getTestFolder();
+  const appName = getUniqueAppName();
+  const subscription = getSubscriptionId();
+  const projectPath = path.resolve(testFolder, appName);
+  const envName = environmentManager.getDefaultEnvName();
+
+  const env = Object.assign({}, process.env);
+  env["TEAMSFX_CONFIG_UNIFY"] = "true";
+  env["BOT_NOTIFICATION_ENABLED"] = "true";
+  env["TEAMSFX_TEMPLATE_PRERELEASE"] = "alpha";
+
+  it("Provision Resource: app service hosted notification", async function () {
```

Comment:
Why not specifying the case number id in `it()`?

Reply:
The case id will be updated after the feature flag is removed and these cases will be added to the test plan.
I will update the test case in the future.